### DISCUSSION TITLE
Add 'find' option to mobile version

### DISCUSSION THREAD

**Discussion by yashwason** [outline/outline#6365] [Ideas]
Outline has a great `find` functionality that works on desktop via the `ctrl + f` shortcut.

This feature should also be available when viewing documents in mobile. The natural place to add the CTA would be the options menu that opens when we click the top-right menu button that exists in document mobile view.

Use case: I was recently viewing a document on mobile and needed to jump to the area where a certain keyword was written. Couldn't do it on mobile. And the place where I naturally checked for the `find` option was in the top-right options menu.

**Comment by tommoor** [comment]
Hi Yash, mobile browsers also have find available in the "share" menu – weird location, I know.

**Reply by yashwason** [reply]
I can't find it? I'm on a self-hosted instance of v0.74.0

https://github.com/outline/outline/assets/37631045/b71e6c7b-fc0d-40c7-8a0a-964092e5fb56

**Comment by tommoor** [comment]
Ah, I was referring to this browser menu which I suppose does not appear in
the PWA. Adding find in the doc menu is easy enough 👍

![image](https://github.com/outline/outline/assets/380914/27506e31-0860-4e27-b212-d3c42393f850)

**Comment by yashwason** [comment]
Great!

https://github.com/outline/outline/issues/6366

I do use the browser's native find feature, but leveraging Outline's own find feature would be better UX for all users, especially those who aren't so inclined. Believe me most people aren't aware of the find option in mobile browsers.
